client/src/pages/Login/Login.tsx

```diff
@@ -39,22 +38,17 @@ export default function Login(): JSX.Element {
   return (
     <Grid container component="main" className={classes.root}>
       <CssBaseline />
-      <Grid className={classes.paper} item component={Paper} square>
-        <Box className={classes.authWrapper}>
-          <AuthHeader linkTo="/signup" asideText="Don't have an account?" btnText="Create account" />
-          <Box width="100%" maxWidth={450} p={3} alignSelf="center" component={Paper}>
-            <Grid container>
-              <Grid item xs>
-                <Typography className={classes.welcome} component="h1" variant="h5">
-                  Welcome back!
-                </Typography>
-              </Grid>
-            </Grid>
-            <LoginForm handleSubmit={handleSubmit} />
-          </Box>
-          <Box p={1} alignSelf="center" />
-        </Box>
-      </Grid>
+      <NavBar />
+      <Paper className={classes.paper} elevation={4}>
+        <Grid container>
+          <Grid item xs>
```

Comment:
I would recommend using a `<Box>` component with `display` prop set to `flex`, `justifyContent` and `alignItems` props set to `center` for a centering container ([Material-UI docs for all the flexbox props](https://material-ui.com/system/flexbox/#api)). You could even refactor this into a re-usable component called `CenteringBox`! :D

Reply:
I will implement a component for centering!